adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java

```diff
@@ -258,6 +262,34 @@ void removeTokenCacheItem(final TokenCacheItem tokenCacheItem, final String reso
         Telemetry.getInstance().stopEvent(mTelemetryRequestId, cacheEvent,
                 EventStrings.TOKEN_CACHE_DELETE);
     }
+
+    boolean isMultipleRTsMatchingGivenAppAndResource(final String clientId, final String resource) {
```

Comment:
I think we should look at merging these 2 functions together, pass null as the resource when you don't need resource.